addons/pos_coupon/models/coupon_program.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# NOTE Use black to automatically format this code.
+
+from odoo import api, fields, models, _
+
+import ast
+
+class CouponProgram(models.Model):
+    _inherit = "coupon.program"
+
+    pos_config_ids = fields.Many2many(
+        "pos.config",
+        relation="program_pos_config_rel",
```

Comment:
check if relation is sill correct, or can be removed with the changes on pos config